hedera-node/src/main/resources/feeSchedules.json

```diff
@@ -1297,7 +1339,44 @@
           "min" : 0,
           "max" : 1000000000000000
         }
-      },{
+      }, {
+        "nodedata" : {
```

Comment:
Hmm...I don't think `TokenAccountWipe` should still need a `DEFAULT` fee schedule entry; will probably remove this.

Reply:
Removed.